Libraries/LibGUI/CppLexer.cpp

```diff
@@ -151,7 +151,7 @@ static bool is_keyword(const StringView& string)
 
 static bool is_known_type(const StringView& string)
 {
-    static HashTable<String> types;
+    static HashTable<String> types(101);
```

Comment:
Please find a way to do this that doesn't require hard-coding the exact number of upcoming entries.
You could use a `constexpr` array of strings for example.